### PR TITLE
Upgrade freetype port from 2.6 to 2.13.3

### PR DESCRIPTION
**Pull request by sbc100**
Also, switch from a fork to using the upstream github repo.  We had a single patch in our fork:

https://github.com/emscripten-ports/FreeType/commit/40a760c963bc2b76575918ff91a515c8474db4e0

However, this was upstreamed in:

https://github.com/freetype/freetype/commit/2f4b740ce435bc1ad5ef8570bb91ab7cd5682720

This change is based on #18088.

Fixes: #22571

### CODE REVIEW

File: test/test_freetype.out [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
-+**+   *****   +
-+***  +**+**+  *
- ***+ ***+**+ +*
- +**+ *** *** +*
- +**++**+ +**+**
-  ***+**+ +**+**
-  ******   *****
-  +****+   +****
-  +****+   +****
-   ****     ****
```
Comment (sbc100) [inline test/test_freetype.out:16]: It looks like the rendering is slightly thinner with the new version.. do you think this kind of change is normal/acceptable when upgrading freetype.

Comment (kripken) [reply test/test_freetype.out:16]: Yeah, it's intentional on the FT side, I assume.

**Review by kripken — APPROVED** [review]

**Comment by sbc100** [comment]
reftests updated.  PTAL.

**Review by kripken — APPROVED** [review]
Great!

**Comment by dschuff** [comment]
This seems to be failing on the Chromium Windows bot: https://ci.chromium.org/ui/p/emscripten-releases/builders/try/win/b8736319351635088817/overview

I haven't tried to repro locally, but let me know if you need me to; i do have a windows cloudtop.

**Comment by sbc100** [comment]
> This seems to be failing on the Chromium Windows bot: https://ci.chromium.org/ui/p/emscripten-releases/builders/try/win/b8736319351635088817/overview
> 
> I haven't tried to repro locally, but let me know if you need me to; i do have a windows cloudtop.

Thanks, I'll take a look.

**Pull request merged**